shipyard_blueprints/rudderstack/shipyard_rudderstack/cli/trigger_sync_cli.py

```diff
@@ -1,16 +1,18 @@
 import argparse
+import time
 import sys
-import requests
-import shipyard_utils as shipyard
+
 from shipyard_rudderstack import RudderStackClient
+from shipyard_templates import ExitCodeException
 
 
 def get_args():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--access-token', dest='access_token', required=True)
-    parser.add_argument('--source-id', dest='source_id', required=True)
-    args = parser.parse_args()
-    return args
+    parser.add_argument("--access-token", dest="access_token", required=True)
+    parser.add_argument("--source-id", dest="source_id", required=True)
```

Comment:
Let's get in the habit of adding types to the CLI args. I think that will help as we begin to improve the `sanity-loader` to be able to populate the datatype in Sanity

Reply:
types other than string on the cli level tend to through errors in the application